web/templates/search_description.hbs

```diff
@@ -3,17 +3,21 @@
     {{#if (eq this.type "plain_text")~}}
         {{~this.content~}}
     {{else if (eq this.type "channel_topic")}}
-        {{~!-- squash whitespace --~}}
-        channel {{this.channel}} > {{this.topic}}
-        {{~!-- squash whitespace --~}}
+        {{~#if is_empty_string_topic~}}
+        channel {{this.channel}} > <span class="empty-topic-display">{{this.topic_display_name}}</span>
+        {{~else~}}
+        channel {{this.channel}} > {{this.topic_display_name}}
+        {{~/if~}}
     {{else if (eq this.type "invalid_has")}}
         {{~!-- squash whitespace --~}}
         invalid {{this.operand}} operand for has operator
         {{~!-- squash whitespace --~}}
     {{else if (eq this.type "prefix_for_operator")}}
-        {{~!-- squash whitespace --~}}
+        {{~#if is_empty_string_topic~}}
+        {{this.prefix_for_operator}} <span class="empty-topic-display">{{this.operand}}</span>{{#if (or (eq this.operand "link") (eq this.operand "image") (eq this.operand "attachment") (eq this.operand "reaction"))}}s{{/if}}
+        {{~ else ~}}
         {{this.prefix_for_operator}} {{this.operand}}{{#if (or (eq this.operand "link") (eq this.operand "image") (eq this.operand "attachment") (eq this.operand "reaction"))}}s{{/if}}
```

Comment:
The reason I didn't prefer 
```hbs
{{this.prefix_for_operator}} <span {{#if is_empty_string_topic}}class="empty-topic-display"{{/if}}>{{this.operand}}</span>{{#if (or (eq this.operand "link") (eq this.operand "image") (eq this.operand "attachment") (eq this.operand "reaction"))}}s{{/if}}
```

Because this adds a lot of churn to add `<span>` to a lot of test cases. And practically too, we don't need the `<span>` tag around operands (although it's noop) except the one case.